### ISSUE TITLE
DataView: CommandButton stops working with Panel `collapsed` property

### ISSUE THREAD

**Issue by stolp** [primefaces/primefaces#12240]
### Describe the bug

After paging a DataView, the command buttons in the paginatorTopRight facet stop triggering actions.

Only reloading the page via F5 remedies that situation.

### Reproducer

See attached reproducer.
[primefaces-test.zip](https://github.com/user-attachments/files/16083108/primefaces-test.zip)

### Expected behavior

The button action should always be triggered.

### PrimeFaces edition

Community

### PrimeFaces version

14.0.2

### JSF implementation

MyFaces

### JSF version

2.3-next

### Java version

11

**Comment by melloware** [comment]
@stolp looks like this has always been like this i tested against 12.0.0 and 13.0.0.  The issue seems to be around `collapsed` being an input value of the panels and you do only `partialProcess="@this"`.

here is the network traffic when you press Expand or Collapse

![image](https://github.com/primefaces/primefaces/assets/4399574/26c29718-4d9b-40e2-8bbd-e60b019d8f1f)


here is is on Page notice the `Collapsed` values of the hidden inputs.

![image](https://github.com/primefaces/primefaces/assets/4399574/22590987-73bd-4aa6-a023-8e9f9050b0a3)

**Comment by stolp** [comment]
I agree that we can track back this issue to PF version 10. Regarding your partialProcess="@this" comment, this occurs even without partialProcess="true" so there must be a different reason for this.

**Comment by stolp** [comment]
Ok understood.
